Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -8,10 +8,117 @@
 Shutdown
 #######################
 
-Under construction.
+Stopping Scheduler
+^^^^^^^^^^^^^^^^^^
+
+The Scheduler automatically adds new targets to the queue until the start of nautical morning twilight.
+If there are no targets, the Scheduler will stop adding new ones.
+If you need to stop the Scheduler manually, you can use the ``auxtel/scheduler/stop.py`` script from ScriptQueue without any additional configurations.
+
+See more detailed descriptions on `Scheduler <https://obs-ops.lsst.io/Nighttime-Operations/Scheduler/night-time-operations/index.html>`__.
+
+
+Shutdown AuxTel
+^^^^^^^^^^^^^^^
+
+After stopping scheduler, you can shutdown AuxTel using ``axutel/shutdown.py`` script from ScriptQueue. 
+``shutdown.py`` script does not any configurations. This script includes:
+
```

Comment:
Maybe add a photo of the script queue where we can see the script?

Reply:
I agree with that. Take a screenshot of the script loaded showing that no config is needed.